### PR TITLE
Import/export tweaks

### LINKED ISSUE #1095: [feature request] option for importing zipped scripts "don't overwrite the current GM_setValue values"
I think it would be useful to add an option for importing zipped scripts titled "Don't read the data saved by the script (don't overwrite the current GM_setValue values)" .
Thanks.

### PR DESCRIPTION
**Pull request by tophf**
* "[x] import script data" option - fixes #1095.
* download via Blob in modern FF

### CODE REVIEW

File: src/common/download.js [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
-  document.body.appendChild(a);
   a.href = url;
-  if (name) a.download = name;
-  a.click();
-  setTimeout(() => {
-    document.body.removeChild(a);
-    if (callback) callback();
-  }, 3000);
+  a.download = name || '';
+  a.dispatchEvent(new MouseEvent('click'));
```
Comment (gera2ld) [inline src/common/download.js:5]: Is there any difference between `a.click()` and `a.dispatchEvent(new MouseEvent('click'));`?

Comment (tophf) [reply src/common/download.js:5]: Yep, click() requires adding to DOM while dispatchEvent works in detached node.

Comment (tophf) [reply src/common/download.js:5]: Should I add it as a comment?

Comment (gera2ld) [reply src/common/download.js:5]: It's OK, we can leave it as is.

**Pull request merged**